packages/itwin/map-layers/src/ui/widget/MapLayerActionButtons.tsx

```diff
@@ -1,10 +1,10 @@
+import * as React from "react";
```

Comment:
this is a little gross with the imports above the copyright, you could just nuke all the copyrights or turn off your formatter

Reply:
the settings.json in this repo had the formatter set to format on save, and prettier was a recommended extension to install. I addressed fixed the formatting, but it seems this repo wanted the formatter (as for why it sorted copyrights in between I dunno)